libs/core/src/lib/status-indicator/status-indicator.component.scss

```diff
@@ -0,0 +1,2151 @@
+/*!
+ * Fundamental Library Styles v0.15.1-rc.8
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company.
+ * Licensed under Apache License 2.0 (https://github.com/SAP/fundamental-styles/blob/main/LICENSE)
+ */
+@charset "UTF-8";
+[class*=sap-icon] {
+  font-size: .875rem;
+  font-size: var(--sapFontSize,.875rem);
```

Comment:
Why do you need to include the entire icon library SCSS here? You should `import` the dependent file instead of copy-pasting the entire content.

Reply:
Status indicator is in the version 16 and our current ngx is pointed to 14 version , on migrating the to the latest version will remove the entire content and replace it import statement.